docs/registry/attributes/http.md

```diff
@@ -51,7 +51,8 @@ If the HTTP request method is not known to instrumentation, it MUST set the `htt
 If the HTTP instrumentation could end up converting valid HTTP request methods to `_OTHER`, then it MUST provide a way to override
 the list of known HTTP methods. If this override is done via environment variable, then the environment variable MUST be named
 OTEL_INSTRUMENTATION_HTTP_KNOWN_METHODS and support a comma-separated list of case-sensitive known HTTP methods
-(this list MUST be a full override of the default known method, it is not a list of known methods in addition to the defaults).
+If this override is done via declarative configuration, then the list key MUST be named `known_methods` (under `general`/`http`).
```

Comment:
I think we should include [`instrumentation/development`](https://github.com/open-telemetry/opentelemetry-configuration/blob/d6b127882f41db92b26017c736a52ceb6f2b4451/examples/kitchen-sink.yaml#L855) here? 

plus given it's in development, but appears on the stable attribute we need to express that the name is in development ? 

E.g. like here https://github.com/open-telemetry/semantic-conventions/blob/463b7f5a3d173bd6d0677834a9ef17da1dac7b2d/docs/http/http-spans.md?plain=1#L115